libs/components/molecules/src/lib/header/header.component.scss

```diff
@@ -0,0 +1,50 @@
+:host {
+  display: flex;
+  // flex-direction: row;
+  // align-items: center;
+  // padding: 1rem 2rem;
+  background: #ffffff;
+  border-bottom: 1px solid #e2e7e9;
+  font-family: 'Inter';
```

Comment:
Remove font-family, etc.

Reply:
removing the font-family causes the text to revert back to default settings and not 'Inter'

Reply:
That will automatically be fixed once I've merged my theming changes today, so you can remove this